src/port.js

```diff
@@ -0,0 +1,17 @@
+class Port {
+  constructor(name) {
+      this.name = name
+      this.ships = []
+  }
+  
+  addShip(ship){
+      this.ships.push(ship)
+  }
+
+  removeShip(ship) {
+     const ind = this.ships.indexOf(ship)
```

Comment:
I would either call this variable either `index` or `i`, as that are the conventions for naming an index